tests/AppBundle/Controller/Security/AdherentSecurityControllerTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\AppBundle\Controller\Security;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Tests\AppBundle\Controller\AbstractControllerTest;
+
+class AdherentSecurityControllerTest extends AbstractControllerTest
+{
+    public function testLoginAction()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request(Request::METHOD_GET, '/espace-adherent/connexion');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $client->getResponse());
+        $this->assertCount(1, $crawler->filter('form[name="app_login"]'));
+        $this->assertCount(0, $crawler->filter('.login--box--error'));
+    }
+
+    public function testLoginCheckFailsOnInvalidCredentials()
+    {
+        $client = static::createClient();
+
+        $client->request(Request::METHOD_POST, '/espace-adherent/connexion/check', []);
```

Comment:
IMHO, you should rather submit the form from the login path. to test the whole flow